utils/deprecated.js

```diff
@@ -30,3 +31,24 @@ export function deprecated( ...params ) {
 
 	return originalDeprecated( ...params );
 }
+
+// keycodes
+const wrapKeycodeFunction = ( source, functionName ) => ( ...args ) => {
+	originalDeprecated( `wp.utils.keycodes.${ functionName }`, {
+		version: '3.3',
```

Comment:
Shouldn't this be `3.4`? This will be out in 3.2, so we should deprecate after two versions.